README.md

```diff
@@ -63,12 +63,22 @@ paymentOptions.setPaymentType(PaymentType.ALL);
 paymentOptions.setStoredCardList(paymentMethodItems);
 
 //set merchant customizations such as page title, icon, and footer text (html)
-paymentOptions.setMerchantTitle("Lucy's Shop");
-paymentOptions.setMerchantIcon(R.drawable.your_icon);
+paymentOptions.setMerchantTitle("Lucy's Shop"); // NOT AVAILABLE!
```

Comment:
Android Studio (Arctic Fox) didn't understand `setMerchantTitle` or `setMerchantIcon`, in addition, these don't seem to be in the code either

Reply:
Were you using kotlin?

Reply:
correct, the docs are indeed out of date. Instead on android devs should use setHeader() to achieve the same result.